src/cmap/commands.ts

```diff
@@ -876,4 +875,14 @@ export class BinMsg {
 
     this.parsed = true;
   }
+
+  private _parseBsonSerializationOptions({ enableUtf8Validation }: BSONSerializeOptions): {
```

Comment:
```suggestion
  parseBsonSerializationOptions({ enableUtf8Validation }: BSONSerializeOptions): {
```

Since we're working on an internal class here, everything's inherently private, its stylistic but the access modifier likely would just get in the way of unit tests written in typescript if that ever came about.

Reply:
Fixed!